rrdd/rrdd_ha_stats.ml

```diff
@@ -48,12 +48,12 @@ module Xapi_latency = struct
   let get () =
     Ds.ds_make ~name:"xapi_healthcheck/latency" ~units:"s"
       ~description:"Turn-around time of the latest xapi status monitoring call on the local host"
-      ~value:(Opt.default Rrd.VT_Unknown (Opt.map (fun x -> Rrd.VT_Float x) !raw))
+      ~value:(Option.fold ~none:Rrd.VT_Unknown ~some:rrd_float !raw)
       ~ty:Rrd.Gauge ~default:false ()
 end
 
 let all () =
-  Mutex.execute m (fun _ ->
+  Xapi_stdext_threads.Threadext.Mutex.execute m (fun _ ->
```

Comment:
i don't understand why you have introducted this,

this is also not consistent with `let m = Mutex.create ()` at the top of the file

Reply:
`Mutex.create` does not come from `Threadext`.

This change is to make the origin of `execute` obvious by removing the `open`

Reply:
ah mb, i though `Mutex.execute` was in stdlib...